src/Microsoft.ML.Core/Data/ITransformModel.cs

```diff
@@ -25,6 +25,13 @@ public interface ITransformModel
         /// </summary>
         ISchema InputSchema { get; }
 
+        /// <summary>
+        /// The resulting schema once applied to this model. The <see cref="InputSchema"/> might have
```

Comment:
>The resulting schema once applied to this model [](start = 12, length = 47)

That is not correct. Indeed the point of writing this documentation is to clarify to users that this will *not* necessarily be the schema when applied. :/